docs/tutorial/quickstart.md

```diff
@@ -0,0 +1,355 @@
+6. Fetch the manifest from the repository.
+7. Parse the fetched manifest content and get the layers.
+8. Copy the artifact from the repository.
+
+The complete code is provided at the end of this tutorial.
+
+## Create a folder for your code
+
+Open a command prompt and cd to your home directory.
+
+On Linux or Mac:
+```shell
+cd
+```
+On Windows:
+```shell
+cd %HOMEPATH%
+```
+
+Create a directory for your code.
+```shell
+mkdir oras-go-v2-quickstart
+cd oras-go-v2-quickstart
+```
+Create a module in which you can manage dependencies.
+
+```shell
+go mod init quickstart/oras-go-v2
+go: creating new go.mod: quickstart/oras-go-v2
+```
+
+In your text editor, create a file `main.go` in which to write your code.
+
+## Connect to a remote repository with basic authentication
+
+Paste the following into `main.go` and save the file. This code demonstrates how to use [NewRepository](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/registry/remote#NewRepository) from the [remote](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/registry/remote) package to connect to a remote repository. Basic authentication (using a username and password) is handled by the [auth](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/registry/remote/auth) package. Other authentication methods, such as refresh token authentication, are also supported.
+
+```
+package main
+
+import (
+	"fmt"
+
+	"oras.land/oras-go/v2/registry/remote"
+	"oras.land/oras-go/v2/registry/remote/auth"
+	"oras.land/oras-go/v2/registry/remote/retry"
+)
+
+func main() {
+	// 1. Connect to a remote repository with basic authentication
+	registry := "example.registry.com"
+	repository := "myrepo"
+	repo, err := remote.NewRepository(fmt.Sprintf("%s/%s", registry, repository))
+	if err != nil {
+		panic(err)
+	}
+	// Note: The below code can be omitted if authentication is not required.
+	repo.Client = &auth.Client{
+		Client: retry.DefaultClient,
+		Cache:  auth.NewCache(),
+		Credential: auth.StaticCredential(registry, auth.Credential{
+			Username: "username",
+			Password: "password",
+		}),
+	}
+}
+```
+
+## Show tags in the repository
+The following code snippet uses the [(*Repository) Tags](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/registry/remote#Repository.Tags) method from the [remote](https://pkg.go.dev/oras.land/oras-go/v2@v2.5.0/registry/remote) package to list the tags in the repository. Paste the code into `main.go` after the last section.
+
+```
+// 2. Show the tags in the repository
+ctx := context.Background()
+err = repo.Tags(ctx, "", func(tags []string) error {
+	for _, tag := range tags {
+		fmt.Println(tag)
+	}
+	return nil
+})
+if err != nil {
+	panic(err)
+}
+```
+
+### Run the code
+
+At the command line, use [`go get`](https://golang.org/cmd/go/#hdr-Add_dependencies_to_current_module_and_install_them) to add the `oras.land/oras-go/v2` module as a dependency for your module. Use a dot argument to mean "get dependencies for code in the current directory."
```

Comment:
Instead of `go get`, should you run `go mod tidy` instead as you do have `go.mod`?

Reply:
Change the workflow to "1. go mod init 2. go get oras.land/oras-go/v2 3. create main.go 4. go mod tidy 5. go run ."